manifest.versions.json

```diff
@@ -36,7 +36,7 @@
 
       "nuget|version": "6.6.1",
 
-      "vs|version": "17.6",
+      "vs|version": "17.7",
```

Comment:
Dockerfiles need to be regen'd.

Reply:
The regen doesn't do anything, the ".6"/".7" part of the version doesn't show up in any Dockerfiles.

Reply:
You're right. My bad. It's just used for tests.